packages/slots/components/MainFooter.tsx

```diff
@@ -0,0 +1,38 @@
+import { SlotComponentProps } from '../SlotRenderer';
+import { Container, Link } from '@plone/components';
+import Logo from './Logo';
+
+const Footer = (props: SlotComponentProps) => {
+  const { content, location } = props;
+  const siteActions = content?.['@components']?.actions?.site_actions || [];
+
+  return (
+    <Container layout className="footer">
+      {/* TODO: i18n properly */}
+      <div className="footer-message">
+        The Plone® Open Source CMS/WCM is © 2000-2024 by the Plone Foundation
```

Comment:
WTF is a WCM, other than alphabet soup? mmm... soup...

Can we make the end year dynamic, so we don't have to update this every January 1?

Reply:
@stevepiercy @tkimnguyen Don't pay attention to wording and text details yet. This changes are mainly foundamental, for "make it work", then we will get back to polish these details.

In this case, it's a copy/pasta from somewhere else. This should not be hardcoded, but come from an easy overridable variable instead. Having to override the footer every single time, sucks, and it's been doing it for two decades.